delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutesSpec.scala

```diff
@@ -131,7 +131,7 @@ class BlazegraphViewsRoutesSpec extends BlazegraphViewRoutesFixtures {
         val postRequest = Post("/v1/views/org/proj/indexing-view/sparql", queryEntity).withHeaders(Accept(mediaType))
         val getRequest  = Get(s"/v1/views/org/proj/indexing-view/sparql?query=$encodedQ").withHeaders(Accept(mediaType))
         forAll(List(postRequest, getRequest)) { req =>
-          req ~> asBob ~> routes ~> check {
+          req ~> routes ~> check {
```

Comment:
Since the authorization for querying is part of `BlazegraphViewsQuery`, this test passes as anonymous